litellm/integrations/traceloop.py

```diff
@@ -0,0 +1,7 @@
+import sys
+from traceloop.tracing import Tracer
+
+
+class TraceloopLogger:
+    def __init__(self):
+        self.tracer = Tracer(sys.argv[0])
```

Comment:
do you not need a `def log_event` in traceloop.py ?

Reply:
Nope @ishaan-jaff :)
We're monkey-patching all the calls and log everything asynchronously with OpenTelemetry.